oeps/processes/oep-0001.rst

```diff
@@ -328,7 +328,11 @@ be merged to capture a set of edits, and to make the proposal more visible to
 community comment. From that point, additional pull requests can be opened to
 edit the OEP, until it converges to being either "Accepted" or "Rejected".
 
-If an OEP has Draft or Under Review status and the PR is under review, you can either use the intended final status (e.g. Provisional, Accepted, etc.), or you can clarify both the current and intended status using something like the following: "Under Review (=> Provisional)". Either of these options is especially useful if the merged status is not intended to be Accepted.
+When an OEP merges with any non-final status, use the References section to
+link to a page which gives readers an updated status. For Provisional OEPs which
+need to be implemented, this should be a top-level rollout plan.
```

Comment:
Do we also want an implementation plan page for Accepted OEPs? I think we say Accepted when everyone agrees it's a good idea, and then implementation follows. For that matter, Draft and Under Review could also have plan pages for finishing and implementing the OEP. For example: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3511091214/OEP-57+Update+Game+Plan

Reply:
Frankly yes I would love implementation plans for accepted OEPs as well. One of the use cases in my mind is when my team started getting Hooks and Signals PRs a year after it was approved but had no idea there was still implementation that might affect us. Also of course we ignored the PRs for a month 😮‍💨 before I realized they were part of an OEP, because I have never made it all the way through the OEPs... So yes, I think anything that can improve visibility around "implementation status" for OEPs is helpful!

Reply:
I like this proposal. Here's a suggested reword that would standardize the "key" as well as encourage an implementation plan regardless of status:

> When an OEP PR calls for significant work after it merges, add a link named
> *Follow-up Work* to the *References* section of the OEP header. Use the linked
> page to keep readers up-to-date on the plan for completing and/or implementing
> the proposal. For OEPs merging with the status of Draft or Provisional,
> a *Follow-up Work* link is required.

Reply:
Thanks @kdmccormick I've made that update!